zhaquirks/philips/soc001.py

```diff
@@ -0,0 +1,45 @@
+"""Signify SOC001 device."""
+
+from zigpy import types
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import BinarySensorDeviceClass, ClusterType, QuirkBuilder
+
+
+class PhilipsContactCluster(CustomCluster):
+    """Philips manufacturer specific cluster for contact sensor."""
+
+    cluster_id = 64518  # 0xfc06
+    name = "Philips contact cluster"
+    ep_attribute = "philips_contact_cluster"
+    attributes = {
+        0x0100: ("contact", types.enum8, True),
+        0x0101: ("last_contact_change", types.uint32_t, True),
+        0x0102: ("tamper", types.enum8, True),
+        0x0103: ("last_tamper_change", types.uint32_t, True),
+    }
```

Comment:
Can we use the new-style zigpy `AttributeDefs` here?

```python
    class AttributeDefs(BaseAttributeDefs):
        """Attribute definitions."""

        contact = ZCLAttributeDef(
            id=0x0100,
            type=types.enum8,
            is_manufacturer_specific=True,
        )
        # ... (and so on) 
```

(There are some examples if you search for `AttributeDefs` in this or the zigpy repo)

Reply:
Done here : https://github.com/zigpy/zha-device-handlers/pull/3432/commits/a12a3e08b45e5c7123965f3ea0d3ece9fd98ee4d